tao/BUILD

```diff
@@ -80,6 +94,7 @@ cc_library(
         ],
         exclude = [
             "tao_bridge/**/*test.cc",
+            "tao_bridge/gpu/bace_gemm_thunk.cc",
```

Comment:
Just remove bace code internal code base?

Reply:
I add a macro to not use `bace` related code now.

Reply:
Great